core/src/main/java/org/jruby/runtime/BlockBody.java

```diff
@@ -50,7 +50,7 @@
 public abstract class BlockBody {
 
     protected final Signature signature;
-    protected volatile MethodHandle testBlockBody;
```

Comment:
This is not used but might be later. It is also to avoid recreating the same handle over and over again. It could be made Object and casted elsewhere to avoid the hard reference of MethodHandle.